package.json

```
@@ -15,10 +15,11 @@
     "commit": "npx git-cz",
     "format": "prettier-eslint --write \"src/**/*.ts\" \"test/**/*.ts\"",
     "start": "nest start",
-    "start:repl": "yarn start --entryFile repl",
     "start:dev": "nest start --watch",
     "start:debug": "nest start --debug --watch",
     "start:prod": "NODE_ENV='production' node dist/main",
+    "start:repl": "yarn start --entryFile commands/repl",
+    "generate:swagger": "yarn start --entryFile commands/write-swagger",
```

```suggestion
    "generate:swagger": "POLYMESH_NODE_URL: 'wss://testnet-rpc.polymesh.live' yarn start --entryFile commands/write-swagger",
```

I am thinking this is probably the best stop gap. Downside being testnet being down would block a release, but we can always rerun / manually add if something happens.

---

For now I propose, lets stick to this to verify that the process we created works and once that is done we can setup the chain as well later